### PR TITLE
AWS RDS Baseline

### PR DESCRIPTION
**Pull request by karikarshivani**
All the profile errors have been addressed. Please let me know if I missed anything!

Signed-off-by: karikarshivani <karikarshivani@gmail.com>

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+copyright: MITRE SAF
+copyright_email: saf@groups.mitre.org
+license: Apache-2.0
+summary: "An InSpec Profile to validate the AWS RDS Microsoft SQL Server 2017 CIS Benchmark"
+version: 1.2.0
+inspec_version: ">= 4.0"
+
+depends:
+- name: microsoft-sql-server-2017-cis-baseline
+  url: https://github.com/mitre/microsoft-sql-server-2017-cis-baseline/archive/v1.2.0.tar.gz
```
Comment (karikarshivani) [inline inspec.yml]: I'll update the branch name when the PR for the baseline has been approved and merged. Thanks!

**Review by wdower — APPROVED** [review]
LGTM, thanks!

**Review by ejaronne — CHANGES_REQUESTED** [review]
- update depends reference to main.tar.gz (not 1.2.0)
- update README exec instructions to use main.tar.gz (not master)
- update README "full heimdall server" url to point to heimdall2 repo not old heimdall

**Review by ejaronne — APPROVED** [review]

File: inspec.yml (hunk 1 above)
Comment (karikarshivani) [reply inspec.yml]: Updated the branch name

**Comment by karikarshivani** [comment]
> * update depends reference to main.tar.gz (not 1.2.0)
> * update README exec instructions to use main.tar.gz (not master)
> * update README "full heimdall server" url to point to heimdall2 repo not old heimdall

Thanks for making these updates! I ran the profile - it's ready to be merged!

**Pull request merged**